tests/load_test.py

```diff
@@ -0,0 +1,117 @@
+import argparse
+import time, asyncio
+from openai import AsyncOpenAI, AsyncAzureOpenAI
+import uuid
+import traceback
+import numpy as np
+
+# base_url - litellm proxy endpoint
+# api_key - litellm proxy api-key, is created proxy with auth
+litellm_client = None
+
+
+async def litellm_completion(args, image_url=None):
+    # Your existing code for litellm_completion goes here
+    try:
+        if image_url:
+            messages = [
+                {
+                    "role": "user",
+                    "content": [
+                        {"type": "image_url", "image_url": {"url": image_url}},
+                        {"type": "text", "text": f"Tell me a story about this image."},
+                    ],
+                },
+            ]
+        else:
+            messages = [
+                {"role": "user", "content": f"Tell me a story about this image."}
+            ]
+
+        start = time.time()
+        response = await litellm_client.chat.completions.create(
+            model=args.model,
+            messages=messages,
+            stream=True,
+        )
+        ttft = time.time() - start
+
+        itl_list = []
+        content = ""
+        start = time.time()
+        async for chunk in response:
+            if chunk.choices[0].delta.content:
+                content += chunk.choices[0].delta.content
+                itl_list.append(time.time() - start)
+                start = time.time()
+
+        return content, ttft, itl_list
+
+    except Exception as e:
+        # If there's an exception, log the error message
+        print(e)
+        with open("error_log.txt", "a") as error_log:
+            error_log.write(f"Error during completion: {str(e)}\n")
+        return str(e)
+
+
+async def main(args):
+    n = 50  # Total number of tasks
+    batch_size = args.req_per_sec  # Requests per second
+    start = time.time()
+
+    all_tasks = []
+    for i in range(0, n, batch_size):
+        batch = range(i, min(i + batch_size, n))
+        for _ in batch:
+            if args.include_image:
+                # Generate a random dimension for the image
+                y_dimension = np.random.randint(100, 1025)
```

Comment:
do you think we should use the same size image to measure latency more consistently?

Reply:
This is a good point but it also depends on the goal of our analysis. One thing to think about is the Phi-3.5 bug which was due to the model not being able to handle a certain resolution, which is why randomizing the input and feeding it in could be useful. But yes, for better determinism so we can compare across models, we should make this constant.

Reply:
maybe we can add an argument say `--random-size` then we randomize the input size

Reply:
agreed. added.